### ISSUE TITLE
qa should support separate master_netdev

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#393]
Originally reported of Google Code with ID 337.

```
Nowadays we like to have master_netdev separate from the instances' bridge.
It would be nice if qa would support these two being separate. As an extra bonus full nicparams specification (type and link) would be great, as this would allow to qa eg. with openvswitch and such :)

Thanks,

Guido
```
Originally added on 2012-12-20 12:18:35 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-12-20 12:18:55 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-12-20 12:21:08 +0000 UTC.

Added Labels: Type-Enhancement Priority-Medium
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 3601d48
Author: Michael Hanselmann <hansmi@google.com>
Date:   Thu Dec 20 15:46:06 2012 +0100

    QA: Support master-netdev and default NIC parameters
    
    Requested in issue 337.
    
    The parameter “bridge” was not documented and is therefore silently
    replaced with “master-netdev”. A note is added to “qa-sample.json”
    describing how comments work.
    
    Signed-off-by: Michael Hanselmann <hansmi@google.com>
    Reviewed-by: Iustin Pop <iustin@google.com>

```
Originally added on 2012-12-21 15:49:44 +0000 UTC.

Mentioned Issue with GoogleCode ID 337 has been migrated to GitHub with ID #393.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
